tests/test_simulate.py

```diff
@@ -0,0 +1,167 @@
+"""Tests for petab/simulate.py."""
+import functools
+import numpy as np
+import os
+import pandas as pd
+import pytest
+import scipy.stats
+from typing import Callable
+
+import petab
+from petab.C import MEASUREMENT
+
+
+class TestSimulator(petab.simulate.Simulator):
+    """Dummy simulator."""
+
+    __test__ = False
+
+    def simulate_without_noise(self) -> pd.DataFrame:
+        """Dummy simulation."""
+        return self.petab_problem.measurement_df
+
+
+@pytest.fixture
+def petab_problem() -> petab.Problem:
+    """Create a PEtab problem for use in tests."""
+    petab_yaml_path = 'doc/example/example_Fujita/Fujita.yaml'
```

Comment:
Would be good to use absolute paths to allow invoking pytest from any directory. (also below)